src/Parser/HavocDemonHunter/Modules/Statistics/Spells/Nemesis.js

```diff
@@ -9,23 +12,30 @@ import { formatDuration } from 'common/format';
 import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
 
 class Nemesis extends Module {
+  static dependencies = {
+    combatants: Combatants,
+    enemies: Enemies,
+  };
+
+  on_initialized() {
+    if (!this.owner.error) {
+      this.active = this.combatants.selected.hasTalent(SPELLS.NEMESIS_TALENT.id);
+    }
+  }
 
   statistic() {
 
-    const nemesisUptime = this.owner.modules.enemies.getBuffUptime(SPELLS.NEMESIS_TALENT.id);
+    const nemesisUptime = this.enemies.getBuffUptime(SPELLS.NEMESIS_TALENT.id);
     const nemesisUptimePercentage = nemesisUptime / this.owner.fightDuration;
 
-    if(nemesisUptime > 0)
-    {
     return (
       <StatisticBox
         icon={<Icon icon="ability_warlock_improveddemonictactics" alt="Nemesis" />}
```

Comment:
Please use `SpellIcon` here to users can consistently hover the icon for tooltips about the spells to learn more about them.